lld/ELF/CMakeLists.txt

```diff
@@ -72,6 +73,7 @@ add_lld_library(lldELF
   Object
   Option
   Passes
+  ProfileData
```

Comment:
Now that I think about it more, I think we only need this dependency because we use `InstrProfReader::`. But this is only used in `lld/Common`, so we might not need this explicit dependency here. Same for `MachO/CMakeLists.txt`, I think `ProfileData` can be removed there.